YOUR_ADMIN/includes/classes/observers/CustomerTaxExemptAdminObserver.php

```diff
@@ -63,7 +63,11 @@ public function update(&$class, $eventID, $p1, &$p2, &$p3, &$p4, &$p5, &$p6)
                           LIMIT 1"
                     );
                     $customers_tax_exempt = ($current_value->EOF) ? '' : $current_value->fields['customers_tax_exempt'];
-                    $field_value = htmlspecialchars($customers_tax_exempt, ENT_COMPAT, CHARSET, true);
+                    if (!empty($customers_tax_exempt)) { 
+                        $field_value = htmlspecialchars($customers_tax_exempt, ENT_COMPAT, CHARSET, true);
+                    } else {
+                        $field_value = ''; 
+                    }
```

Comment:
It'd be simpler to simply cast the database value (which could be `NULL`) to a string.

Reply:
```
$field_value = htmlspecialchars((string)$customers_tax_exempt, ENT_COMPAT, CHARSET, true);
```
like that?

Reply:
That works!